dpd/src/table/route_ipv4.rs

```diff
@@ -30,7 +30,7 @@ struct IndexKey {
 
 // Route entries stored in the index->route_data table
 #[derive(ActionParse, Debug)]
-enum RouteAction {
+pub(crate) enum RouteAction {
```

Comment:
I know it leads to a little code duplication, but I'd like to keep the table keys private to their modules.